### PR TITLE
dev

### LINKED ISSUE #85: feature request: adding support for Tukey's IQR method for outlier detection
Maybe this can be supported in `check_outliers`, of course, only for dataframe objects? 
http://colingorrie.github.io/outlier-detection.html#iqr-method

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
close #85

@IndrajeetPatil What do you think?

### CODE REVIEW

File: R/check_outliers.R [hunk 1]
```diff
@@ -306,6 +352,67 @@ check_outliers.data.frame <- function(x, method = "mahalanobis", threshold = NUL
...
+#' @importFrom stats IQR
+.check_outliers_iqr <- function(x, threshold = 1.5, method = "tukey") {
+
+  d <- data.frame(Obs = 1:nrow(as.data.frame(x)))
+  for(col in 1:ncol(as.data.frame(x))){
+
+    v <- x[, col]
+
+    if(method == "tukey"){
+      iqr <- quantile(v, 0.75) - quantile(v, 0.25)
```
Comment (strengejacke) [inline R/check_outliers.R]: don't forget the `stats::`, for IQR as well.

**Comment by DominiqueMakowski** [comment]
I wonder if we should adapt this documentation as a vignette, as it is quite comprehensive. But in general, performance needs some love vignette-wise...

**Comment by strengejacke** [comment]
> But in general, performance needs some love vignette-wise...

Yes, [true](https://easystats.github.io/performance/articles/r2.html)...

**Comment by IndrajeetPatil** [comment]
The code looks good to me, but I don't have a good sense of what the output looks like.

Can you please post an example where this function is used with the IQR method?

**Comment by DominiqueMakowski** [comment]
``` r
library(performance)
check_outliers(mtcars$wt, method = "iqr")
#> Warning: 3 outliers detected (cases 15, 16, 17).
as.numeric(check_outliers(mtcars$wt, method = "iqr"))
#>  [1] 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
```

<sup>Created on 2019-09-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by IndrajeetPatil** [comment]
Cool, looks good to me! Thanks.

**Comment by DominiqueMakowski** [comment]
I'll let @strengejacke merge I know he likes it

**Comment by strengejacke** [comment]
> I'll let @strengejacke merge I know he likes it

download

**Pull request merged**